### PR TITLE
add pykriging

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes this is a noarch recipe can I please get review/merge?

File: recipes/pykriging/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - dill
+    - matplotlib
+    - inspyred
+
+test:
+  imports:
+    - pyKriging
+
+about:
+  home: http://github.com/capulson/pyKriging
```
Comment (synapticarbors) [inline recipes/pykriging/meta.yaml]: Url incorrect:

https://github.com/capaulson/pyKriging

**Comment by CJ-Wright** [comment]
@synapticarbors thank you very much, fixed!

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**